Assets/MixedRealityToolkit/_Core/Resources/Textures/Oculus lineart attribution.txt

```diff
@@ -0,0 +1,5 @@
+You may use these images solely for referring to the corresponding product in your video game or VR experience (including manuals for users). Otherwise, you may not use these images, or any Oculus trademarks, logos or other intellectual property, including but not limited to use on merchandise or other product such as clothing, hats, or mugs. Do not use the Oculus images in a way that implies a partnership, sponsorship or endorsement; or features Oculus on materials associated with pornography, illegal activities, or other materials that violate Oculus Terms.
```

Comment:
Also on that note, where did the rest of the images come from and what are their licenses?

Reply:
I made all the images, but just in case there was ambiguity I added the license.

Reply:
Depends on how technical you wanna get.
I never "drew" anything. 😋

I did process the images and make sure they were in a format that was suitable for use, which involved resizing, and editing the images from their original form. 

I added a 5 px stroke and cleaned up the images where there was errors.